.github/workflows/playwright-tests-qa.yml

```diff
@@ -1,14 +1,13 @@
 name: Playwright Tests on QA
 
 on:
-  push:
-    branches: [main]
-  pull_request:
   workflow_call:
     inputs:
       PLAYWRIGHT_BASE_URL:
         required: false
         type: string
```

Comment:
Similar to the other workflow, I dont think this `workflow_call` block is needed